packagers/npm-global/README.md

```diff
@@ -0,0 +1,18 @@
+# npm package manager
+
+Tests `npm`.
```

Comment:
Missing update on test README

Reply:
Changed it to `Tests global install of `prisma` package with `npm`.`